kombu/transport/redis.py

```diff
@@ -146,8 +146,14 @@ def __init__(self, *args, **kwargs):
     def append(self, message, delivery_tag):
         delivery = message.delivery_info
         EX, RK = delivery['exchange'], delivery['routing_key']
+        if redis.VERSION[0] >= 3:
```

Comment:
Please add a TODO to remove this once we drop support for redis-py 2.x.

Reply:
Done.